nbs/103d_models.gMLP.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #57.            assert seq_len % patch_size == 0, "<code>image_size</code> must be divisibe by <code>patch_size</code>"
</pre><p>There's a small typo here (image_size -&gt; seq_len) :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/timeseriesAI/tsai/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks, fixed in https://github.com/timeseriesAI/tsai/pull/362/commits/d60ddb3aad568fb04bf06de8aba20189ed556cf1 :)